package.json

```diff
@@ -241,7 +241,7 @@
     "prettier": "^2.6.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-query": "^3.39.1",
+    "react-query": "^4.0.0-beta.23",
```

Comment:
It would probably be a good idea (although we can't test with v3 anymore.)
I'm working on #1239 now and will add more tests, so once it's ready, we'll be able to run the new tests.

Reply:
you're replied so soon, are u live in github haha

Reply:
https://github.com/pmndrs/jotai/pull/1239
This is amazing about the enhancement, Thanks for ur work. I'll gladly see how do u solve the problem of error handling, cuz i tried but failed😓